tests/pytorch_tests/model_tests/feature_models/qat_test.py

```diff
@@ -140,14 +143,15 @@ def compare(self, ptq_model, qat_ready_model, qat_finalized_model, input_x=None,
         if self.finalize:
             all_inferable_quantizers = get_all_subclasses(BasePyTorchInferableQuantizer)
             for _, layer in qat_finalized_model.named_children():
-                self.unit_test.assertTrue(isinstance(layer, qi.PytorchQuantizationWrapper))
-                if isinstance(layer.layer, nn.SiLU):
+                if isinstance(layer, DummyPlaceHolder) or isinstance(layer, nn.SiLU):
+                    continue
```

Comment:
This is because we have several types: **PytorchQuantizationWrapper**, **PytorchActivationHolder**" and the testing is per type (and for regular layers we don't have any checks).